tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+"""
+EEG source reconstruction with a template MRI
+=============================================
+
+This tutorial explains how to perform source reconstruction using
+EEG using template MRI subject. We use here the fsaverage brain
+provided by freesurder.
+
+.. important:: Source reconstruction without an individual T1 MRI from the
+               subject will be less accurate. Do not over interpret
+               activity locations which can be off by multiple centimeters.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""  # noqa: E501
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD Style.
+
+import os.path as op
+import mne
+from mne.datasets import sample
+from mne.datasets import eegbci
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+trans_fname = op.join(op.dirname(mne.__file__), "data", "fsaverage",
+                      "fsaverage-trans.fif")
+
+##############################################################################
+# Load the data
+# -------------
+#
+# We use here EEG data from the BCI dataset.
+
+raw_fname, = eegbci.load_data(subject=1, runs=[6])
+raw = mne.io.read_raw_edf(raw_fname, preload=True, stim_channel='auto')
+
+# Clean channel names to be able to use a standard 1005 montage
+ch_names = [c.replace('.', '') for c in raw.ch_names]
+raw.rename_channels({old: new for old, new in zip(raw.ch_names, ch_names)})
+
+# Read and set the EEG electrode locations
+montage = mne.channels.read_montage('standard_1005', ch_names=raw.ch_names,
+                                    transform=True)
+raw.set_montage(montage)
+
+
+subject = 'fsaverage'
+
+# Check that the locations of EEG electrodes is correct with respect to MRI
+mne.viz.plot_alignment(raw.info, subject=subject, eeg="projected",
+                       subjects_dir=subjects_dir, trans=trans_fname)
+
+##############################################################################
+# Setup source space and compute forward
+# --------------------------------------
+
+src = mne.setup_source_space(subject, spacing='oct6',
+                             subjects_dir=subjects_dir, add_dist=False)
+print(src)
+
+mne.viz.plot_bem(subject=subject, subjects_dir=subjects_dir,
+                 brain_surfaces='white', src=src, orientation='coronal')
+
+conductivity = (0.3, 0.006, 0.3)  # for three layers
+model = mne.make_bem_model(subject=subject, ico=4,
```

Comment:
the difference between `make_bem_model` and `make_bem_solution` may be confusing to a newcomer. Maybe add a short comment or just rename the `model` variable to `bem_surfaces`?